### PR TITLE
Fix for the issue #17

### PR DESCRIPTION
**Pull request by timgluz**
PodspecParser was failing due the changes in the specifications - i just updated fixtures files associated to failed testcases. All tests related to PodspecParser are passing now;

All of this wasn't so simple - i was required to fork the [CocoaPods/Core](https://github.com/CocoaPods/Core) project as all of its releases use `activesupport > 3, < 5`, which runned into conflicts with the dependencies in `versioneye-core`. 
My fork just includes `activesupport > 5` and requires  `Ruby > 2.3.1`

ps: this upgrade breaks compability with older `podspecs < 1.0`

### CODE REVIEW

**Comment by reiz** [comment]
@timgluz Look OK so far, but I have to test if this works with all Pods in the CocoaPods repository. If we can't crawl old Pods with this than maybe we have to maintain 2 different CocoaPods crawlers. I can test it either this afternoon or tomorrow.

**Comment by timgluz** [comment]
Sadly, this fix introduces breaking changes - you must include somehow 2different versions of `Cocoapods/Core`, one for old version and another for new; and i'm pretty sure that many projects dont even bother to upgrade;

File: spec/fixtures/files/pod_file/target_example1/Podfile [hunk 1]
```diff
@@ -23,7 +23,6 @@ target :lite do
 end
 
 target :demo do
-    link_with 'app-demo'
```
Comment (reiz) [inline spec/fixtures/files/pod_file/target_example1/Podfile]: @timgluz Why did you remove this? Can it not be parsed that way with the newest CocoaPods Gem?

File: spec/fixtures/files/pod_file/target_example1/Podfile [hunk 2]
```diff
@@ -11,7 +11,7 @@ target :default do
 end
 
 target :lite do
-    link_with 'app-lite'
```
Comment (timgluz) [inline spec/fixtures/files/pod_file/target_example1/Podfile]: Yes, newest parser fails to even parse the file as `link_with` is now deprecated; source: [https://guides.cocoapods.org/syntax/podfile.html#link_with](https://guides.cocoapods.org/syntax/podfile.html#link_with)
